### ISSUE TITLE
Create new id for `local` in POSIX script

### ISSUE THREAD

**Issue by go2null** [koalaman/shellcheck#1727]
### For new checks and feature suggestions
- [ ] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related
- [x] I didn't find this suggestion.

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/sh 

a_function() {
  local var
}
```

#### Here's what shellcheck currently says:
*In POSIX sh, 'local' is undefined. [SC2039] [sh/shellcheck]*

#### Here's what I wanted or expected to see:

SC2039 is a generic handler.  I would like a specific id for this case so that I can disable it globally in a script.

**Comment by slycordinator** [comment]
It seems that this has been fixed/implemented, since now shellcheck issues "individual SC3xxx warnings" and no longer uses the SC2039 warning mentioned.

For instance, local triggers SC3043 separately from any other warning for POSIX sh.

**Comment by go2null** [comment]
fixed with [SC3043](https://github.com/koalaman/shellcheck/wiki/SC3043)
